fix: add warning for form action responses lost without SSR (#12063)
sveltejs/kit: .changeset/many-cats-breathe.md

---
"@sveltejs/kit": patch
---

> fix: Add warning for form action responses lost without SSR

Reviewer: ```suggestion
fix: warn when form action responses are lost because SSR is off
```
